kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -4,14 +4,39 @@
 """
 from __future__ import absolute_import, print_function, unicode_literals
 
+import json
+
 from django import template
-from kolibri.core.hooks import NavigationHook
```

Comment:
What's the difference between NavigationHook and UserNavigationHook? Why did we keep the former?